common/app/rendering/core/RenderingActor.scala

```diff
@@ -1,21 +1,30 @@
 package rendering.core
 
 import akka.actor.Actor
-import model.ApplicationContext
+import common.StopWatch
 import rendering.Renderable
 import scala.util.Try
 
-case class Rendering(renderable: Renderable, ac: ApplicationContext)
+case class Rendering(renderable: Renderable, forceReload: Boolean = false)
 
 case class RenderingException(error: String) extends RuntimeException(error)
 
 class RenderingActor extends Actor with JavascriptRendering {
 
   override def javascriptFile: String = "ui/dist/ui.bundle.server.js"
 
+  override def preStart: Unit = {
+    // Warming up the script engine
+    val stopWatch = new StopWatch
+    1.to(25).foreach(_ => render(None))
```

Comment:
We would need to tweak this number. 25 is not enough iterations for the script engine to fully warmup apparently but it would help going though the first few execution that are quite slow.
I want to know more about how long the warmup take in PROD before to increase this value